creator-node/src/middlewares.js

```diff
@@ -15,6 +15,7 @@ const { hasEnoughStorageSpace } = require('./fileManager')
 const { getMonitors, MONITORS } = require('./monitors/monitors')
 const { verifyRequesterIsValidSP } = require('./apiSigning')
 const BlacklistManager = require('./blacklistManager')
+const StateMachineManager = require('./services/stateMachineManager')
```

Comment:
instead of importing this it should be taken from serviceRegistry because serviceRegistry already initted it